exercises/practice/alphametics/.meta/config.json

```
@@ -0,0 +1,15 @@
+{
+  "authors": [],
```

You are definitely the author of this exercise (you'll get reputation for that and credited on the website):

```suggestion
  "authors": ["chip"],
```